File: myCTCA Mobile/iOS/src/myctca/Models/Vitals.swift
//
//  Vitals.swift
//  myctca
//
//  Created by Tomack, Barry on 2/27/18.
//  Copyright © 2018 CTCA. All rights reserved.
//

import Foundation

class VitalsInfo: Codable {
    var enteredDateString: String = ""
    var details = [Vitals]()
    
    enum CodingKeys: String, CodingKey {
        case details
        case enteredDateString = "enteredDate"
    }
}

class Vitals: Codable {
    var observationItem: String? = ""
    var value: String = ""
    var displaySequence: Int? = 0

    var enteredDate = Date()

    enum CodingKeys: String, CodingKey {
        case observationItem, value, displaySequence
    }
}

File: myCTCA Mobile/Android/src/myctca/app/src/main/java/com/myctca/model/Vitals.java
package com.myctca.model;

/**
 * Created by tomackb on 2/27/18.
 */

public class Vitals extends HealthHistoryDoc {

    private static final String TAG = "VITALS";
    private String observationItem;
    private String value;
    private int displaySequence;

    public String getObservationItem() {
        return observationItem;
    }

    public String getValue() {
        return value;
    }

    public int getDisplaySequence() {
        return displaySequence;
    }
}
